engine/src/flutter/runtime/runtime_controller.h

```diff
@@ -779,6 +787,8 @@ class RuntimeController : public PlatformConfigurationClient,
   double GetScaledFontSize(double unscaled_font_size,
                            int configuration_id) const override;
 
+  friend class testing::RuntimeControllerTester;
```

Comment:
It would be really nice to avoid adding test code to release code. What is it that's needed from this class by `RuntimeControllerTester`? Could that simply be made public instead of giving full access to the internals of this class?